client/src/ui/molecules/language-toggle/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Translation } from "../../../document/types";
+
+import "./index.scss";
+
+export function LanguageToggle({
+  locale,
+  translations,
+}: {
+  locale: string;
+  translations: Translation[];
+}) {
+  function getLink(locale) {
+    if (locale.toLowerCase() !== "en-us") {
+      for (const translation of translations) {
+        if (translation.locale.toLowerCase() === "en-us") {
+          return (
+            <a href={translation.url} className="language-icon default">
+              <span className="show-desktop">View in</span> English
+            </a>
+          );
+        }
+        return null;
```

Comment:
This is wrong. If the first item in the `translations` array is not en-US, the whole `getLink()` will return `null`.